ui/src/common/loginToContinueModal.scss

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+@media screen and (max-width:575.98px) {
+  div[role='dialog'] {
+    z-index: 1100;
+  }
+}
```

Comment:
In fact, this should be regarded as a global bug, because the z-index attribute of tootip defined in bootstrap is higher than that of modal, so I think we should increase the z-index of modal or lower the z-index of tooltip.

bootstrap defined CSS

![image](https://github.com/apache/incubator-answer/assets/30591202/d975930d-0be4-4c58-af98-bef0591350d4)

You can define variables in the `src/common/_variable.scss` directory

![image](https://github.com/apache/incubator-answer/assets/30591202/59464760-f77e-463b-9116-bf23e3941e72)